openfe/utils/visualization.py

```diff
@@ -227,13 +245,18 @@ def draw_one_molecule_mapping(mol1_to_mol2, mol1, mol2, d2d=None):
     uniques = _get_unique_bonds_and_atoms(mol1_to_mol2, mol1, mol2)
     atoms_list = [uniques["atoms"] | uniques["elements"]]
     bonds_list = [uniques["bonds"]]
-    red = (220/255, 50/255, 32/255, 1.)
-    blue = (0, 90/255, 181/255, 1.)
+    red = (220 / 255, 50 / 255, 32 / 255, 1.0)
+    blue = (0, 90 / 255, 181 / 255, 1.0)
 
     atom_colors = [{at: blue for at in uniques["elements"]}]
 
-    return _draw_molecules(d2d, [mol1], atoms_list, bonds_list, atom_colors,
-                           red)
+    return _draw_molecules(d2d,
+                           [mol1],
+                           atoms_list,
+                           bonds_list,
+                           atom_colors,
+                           red
+                           )
```

Comment:
I'm not going to block over something like this.,.. but this is the kind of autoformatter shenanigans that makes me hate them.

Reply:
If you are going to format it like this, at least add the trailing `,` after red so that if we ever add something to the list of args then the diff is smaller